### PR TITLE
chore: authenticate with npm before publishing package changes

### PR DESCRIPTION
**Pull request by martin3walker**

### CODE REVIEW

File: .circleci/config.yml
Comment (martin3walker) [inline .circleci/config.yml:115]: This should be already done in vault.

**Review by andipaetzold — APPROVED** [review]
Let's wait for tolkien's approval to make sure we didn't mess things up

**Review by 2wce — APPROVED** [review]
LGTM

File: .circleci/config.yml
Comment (chrishelgert) [inline .circleci/config.yml:29]: As it is now only used within the release step, can we remove it and use the command directly?

**Review by chrishelgert — APPROVED** [review]

File: .circleci/config.yml
Comment (denkristoffer) [reply .circleci/config.yml:115]: You need the `vault/configure-lerna` step

File: .circleci/config.yml
Comment (martin3walker) [inline .circleci/config.yml:109]: @chrishelgert does this look alright?

File: .circleci/config.yml
Comment (martin3walker) [inline .circleci/config.yml:108]: @denkristoffer Is this correct?

**Review by chrishelgert — APPROVED** [review]

**Pull request merged**
